### ISSUE TITLE
Failed to install wrapt>=1.11.1 from tensorflow

### ISSUE THREAD

**Issue by jdom05** [chaquo/chaquopy#582]
#### Chaquopy version

The version of Chaquopy is 7.0.3 (com.chaquo.python:gradle:7.0.3).

#### Relevant information

macOS 12.1
Python version: 3.8.1
Android Grandle Plugin Version: 3.6.0
Grandle Version: 5.6.4

```
python {
            buildPython "/Library/Frameworks/Python.framework/Versions/3.8/bin/python3"
            pip{

                install "numpy"
                install "scikit-learn"
                install "tensorflow"

            }
}

ndk {
            abiFilters "x86", "x86_64", "arm64-v8a", "armeabi-v7a" // 345 mb
}
```

#### Describe your issue

```
Collecting wrapt>=1.11.1 (from tensorflow)
  Using cached https://files.pythonhosted.org/packages/eb/f6/d81ccf43ac2a3c80ddb6647653ac8b53ce2d65796029369923be06b815b8/wrapt-1.13.3.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/q3/d8lxpwm14fg6qr98w0d0sw1c0000gn/T/pip-install-2gv2o8ca/wrapt/setup.py", line 41, in <module>
        setuptools.setup(
      File "/Users/.../app/build/generated/python/bp.zip/setuptools/__init__.py", line 129, in setup
      File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/distutils/core.py", line 121, in setup
        dist.parse_config_files()
      File "/Users/.../app/build/generated/python/bp.zip/setuptools/dist.py", line 484, in parse_config_files
      File "/Users/.../app/build/generated/python/bp.zip/setuptools/config.py", line 106, in parse_configuration
      File "/Users/.../app/build/generated/python/bp.zip/setuptools/config.py", line 382, in parse
      File "/Users/.../app/build/generated/python/bp.zip/setuptools/config.py", line 355, in parse_section
      File "/Users/.../app/build/generated/python/bp.zip/setuptools/config.py", line 173, in __setitem__
      File "/Users/.../app/build/generated/python/bp.zip/setuptools/config.py", line 430, in _parse_version
      File "/Users/.../app/build/generated/python/bp.zip/setuptools/config.py", line 305, in _parse_attr
      File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/importlib/__init__.py", line 127, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
      File "<frozen importlib._bootstrap>", line 991, in _find_and_load
      File "<frozen importlib._bootstrap>", line 973, in _find_and_load_unlocked
    ModuleNotFoundError: No module named 'wrapt'
--------------------------------------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/q3/d8lxpwm14fg6qr98w0d0sw1c0000gn/T/pip-install-2gv2o8ca/wrapt/
ERROR: Failed to install wrapt>=1.11.1 from https://files.pythonhosted.org/packages/eb/f6/d81ccf43ac2a3c80ddb6647653ac8b53ce2d65796029369923be06b815b8/wrapt-1.13.3.tar.gz (from tensorflow).
For assistance, please raise an issue at https://github.com/chaquo/chaquopy/issues.
Chaquopy: Exit status 1
Failed to install wrapt>=1.11.1 from https://files.pythonhosted.org/packages/eb/f6/d81ccf43ac2a3c80ddb6647653ac8b53ce2d65796029369923be06b815b8/wrapt-1.13.3.tar.gz (from tensorflow).

> Task :app:generateDebugPythonRequirements FAILED
```

Thanks!!

**Comment by mhsmith** [comment]
This issue was fixed in Chaquopy 9.1.0, but if possible we recommend you update to the current stable version, 11.0.0.

**Comment by mhsmith** [comment]
Alternatively, you could work around the problem by forcing an older version of wrapt. For example, if you remember that your app last built successfully in June 2021, then you can check the [wrapt version history](https://pypi.org/project/wrapt/#history) and you'll see that the last stable version released before that date was 1.12.1. So add "wrapt==1.12.1" to the pip section of your build.gradle file, and it'll use that version rather than the newest available one.

For native packages like NumPy you should check the [Chaquopy package repository](https://chaquo.com/pypi-7.0/) first, as that will take priority over PyPI.

In fact, it's a good idea to do this with all packages to prevent your app from changing when new versions of packages are released. Once you have a working build, you can get the version numbers from the build log, and then list them in your build.gradle file, or a requirements file referenced with ["-r" syntax](https://chaquo.com/chaquopy/doc/current/android.html#requirements).
